js/src/cli/apps.ts

```diff
@@ -28,13 +31,18 @@ export default class AppsCommand {
     new AppUpdateCommand(command);
   }
 
-  private async handleAction(options: { browser: boolean }): Promise<void> {
+  private async handleAction(options: { browser: boolean, enabled: boolean }): Promise<void> {
     getOpenAPIClient();
+    const onlyShowEnabledApps = options.enabled;
+    console.log(onlyShowEnabledApps)
```

Comment:
This console.log statement appears to be a debugging leftover and should be removed from production code.